demos/stacked-bar.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Stacked Bar Chart Demo - Britecharts</title>
+    <link type="text/css" rel="stylesheet" href="styles/britecharts.css" />
+</head>
```

Comment:
yes,I create this branch while 'fix docs merged' PR have not been merged,I forget about it.should remove there lines.